build.assets/Dockerfile-centos7-fips

```diff
@@ -79,6 +84,37 @@ ENV GOEXPERIMENT=boringcrypto \
     GOROOT="/opt/go" \
     PATH="/opt/llvm/bin:$PATH:/opt/go/bin:/go/bin:/go/src/github.com/gravitational/teleport/build"
 
+# Install Rust
```

Comment:
This part is not installing Rust per se. Can we move it lower where the rust is being installed?